src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/Internal/HostTests.cs

```diff
@@ -1382,9 +1445,13 @@ public AsyncThrowingService(Task executeDelayTask)
 
             protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             {
-                await _executeDelayTask;
+                while (true)
+                {
+                    if (_executeDelayTask is { })
```

Comment:
```suggestion
                    if (_executeDelayTask is not null)
```

Reply:
@jaredpar told me that `is { }` was what the hip kids do 🤓.

Reply:
I guess the `dotnet/runtime` kids aren't hip 😞